packages/mask/src/components/InjectedComponents/ProfileCard/ProfileBar.tsx

```diff
@@ -138,7 +138,12 @@ export const ProfileBar = memo<ProfileBarProps>(
                 window.removeEventListener('scroll', closeMenu, false)
             }
         }, [])
-        const selectedAddress = socialAccounts.find((x) => isSameAddress(x.address, address))
+        const selectedAccount = socialAccounts.find((x) => isSameAddress(x.address, address))
+        const { setPair } = ScopedDomainsContainer.useContainer()
+        useEffect(() => {
+            if (!selectedAccount?.address || !selectedAccount.label) return
+            setPair(selectedAccount?.address, selectedAccount?.label)
+        }, [selectedAccount])
```

Comment:
Use a memo stead?

Reply:
What's the point of memo? Need to sync current pair to `ScopedDomainsContainer`

Reply:
It's not a good practice to set state in `useEffect` for such initial values.

```ts
const { setPair } = ScopedDomainsContainer.useContainer([
    selectedAccount,
])
```